testsuite/domain/src/test/resources/host-configs/host-secondary.xml

```diff
@@ -23,7 +23,7 @@
 <host xmlns="urn:jboss:domain:20.0"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="urn:jboss:domain:20.0 wildfly-config_20_0.xsd"
-      name="slave">
+      name="secondary">
```

Comment:
Check HcExtensionAndSubsystemManagementTestCase#checkSlaveReconnected, it looks like it is using a wrong host name